src/lib/ensureArray.test.js

```diff
@@ -0,0 +1,13 @@
+import { ensureArray } from './ensureArray';
+
+describe('ensureArray', () => {
```

Comment:
No need for this `describe`, since the filename's already saying we're testing `ensureArray`

Reply:
Agree. We can build tests without `describe` and using `test` instead of `it`. Looks cleaner.